pysam/libcfaidx.pyx

```diff
@@ -245,7 +245,8 @@ cdef class FastaFile:
               reference=None,
               start=None,
               end=None,
-              region=None):
+              region=None,
+              return_bytes=None):
```

Comment:
Is there any reason why the return_bytes default value is None rather than False?

Reply:
nope, good spot!